sale_coupon_multi_gift/views/sale_coupon_program_views.xml

```diff
@@ -17,11 +17,8 @@
                     <field name="coupon_multi_gift_ids">
                         <tree editable="top">
                             <field name="reward_product_quantity" string="Qty" />
-                            <field name="reward_product_id" string="Product" />
-                            <field
-                                name="reward_product_uom_id"
-                                groups="uom.group_uom"
-                            />
+                            <field name="reward_default_product_id" string="Product" />
```

Comment:
I think we can put it as readonly here in UI, and maybe call the inverse as the onchange for direct UI reflection.

Reply:
Done :+1: